src/models/TerraformPlugin.js

```diff
@@ -14,13 +14,19 @@ import packageInfo from 'package.json';
 class TerraformPlugin extends DefaultPlugin {
   /**
    * Default constructor.
+   *
+   * @param {object} [props={}] - Object that contains all properties to set.
```

Comment:
shouldn't it be `{Object}` ? like `{Function}` under...

Reply:
It's manage by linter and jsdoc, if you set Object we have a linter error. So it's good like that !